doc/null_safety/null_safe_migration.md

```diff
@@ -51,9 +51,11 @@ dart pub global run over_react_codemod:null_safety_migrator_companion --yes-to-a
 ```
 
 This codemod will:
-- Add nullability hints to props/state that are defaulted/initialized in class components.
```

Comment:
Not sure if I should add a note to the prop requiredness codemod section that it also checks if the props are defaults or just let it be assumed that that codemod handles all things props requiredness?

Reply:
Yeah, I think it'd be helpful to mention that, since there are a lot of moving pieces with these codemods.

Reply:
Done! Open to suggestions on wording / format for both of those updates!